_layouts/get-started.html

```diff
@@ -14,7 +14,7 @@ <h2 class="step">Step <span class="step-no">1</span></h2>
       <h3>Get Infinispan</h3>
       <p>Add a username and password, then run the one of the following commands:
         <pre><code>
-podman run -it -e USER="username" -e PASS="password" --net=host quay.io/infinispan/server:12.0
+podman run -it -p 11222:11222 -e USER="username" -e PASS="password" infinispan/server:12.0
```

Comment:
With podman I think you need `--net=host` as well as `-p ...`

Also, we should use `quay.io` for all images.

Reply:
Thanks @ryanemerson yeah I think you're right. I've updated and both commands work for me.